R/getNumSum.R

```diff
@@ -0,0 +1,36 @@
+getNumSum = function(data, features, target) {
+  num.data = subset(data, select = features)
+  #for the next calculations NAs will be removed
```

Comment:
Instead of only mention it as comment, you could writing a warning message?

Reply:
alright will do that